liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppsMojo.java

```diff
@@ -78,11 +83,54 @@ private void installDependencies() throws Exception {
             }
         }
     }
+    
+    private void installProject() throws Exception {
+        if (isSupportedType(project.getPackaging())) {
+            if (looseConfig) {
+                switch(project.getPackaging()) {
+                    case "war":
+                        installLooseConfigApp();
+                        break;
+                    case "liberty-assembly":
+                        File dir = new File(project.getBasedir() + "/src/main/webapp");
+                        if (dir.exists()) {
+                            installLooseConfigApp();
+                        } else {
+                            log.debug("liberty-assembly project does not have soruce code for web project.");
+                        }
+                        break;
+                    default:
+                        //TODO: revise and move message to MvnMessages.properties
+                        log.info("Can not generate loose configuration for Project artifact type. Project aritifact will be installed as is.");
```

Comment:
suggestions for when you revise:
Loose application configuration is not supported for packaging type {0}. The project artifact will be installed as is.